packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -688,7 +688,12 @@ abstract class Element implements BuildContext {
   Widget get widget => _widget;
   Widget _widget;
 
-  bool _active = false;
+  RenderObjectOwner _owner;
+  RenderObjectOwner get owner => _owner;
+  void set owner(RenderObjectOwner value) {
```

Comment:
it's fine if the owner concepts are related, IMHO. Every widget tree is rooted at a render object. They share a pipeline, so it makes sense that the widgets pipeline would be a subclass of the render object one.